backend/src/test/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtilsTest.java

```diff
@@ -0,0 +1,66 @@
+package gov.cdc.usds.simplereport.validators;
+
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.ValueOrError;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.getValue;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateEthnicity;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validatePhoneNumber;
+import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.validateZipCode;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class CsvValidatorUtilsTest {
```

Comment:
I thought for a while about the best way to test this class - there are a _ton_ of functions not being tested here, mostly because I felt it would be repetitive and not terribly useful given that most would be asserting on whether or not an input was in a given set or conformed to a regex, and that's being tested at the FileValidator level instead. But would welcome any thoughts or opinions about the test structure here (and in the other two tests)